barebones/Makefile

```diff
@@ -0,0 +1,23 @@
+#Coremark makefile for BP
+
+RISCV_GCC = riscv64-unknown-elf-gcc -mabi=lp64 -march=rv64ia -mcmodel=medany -fno-builtin-printf
+
+RISCV_LINK = -O2 -static -nostdlib -nostartfiles -lm -lgcc -T test.ld
+
+RISCV_TESTS_DIR = ../../riscv-tests/benchmarks
+
+COMMON_FILES = -I$(RISCV_TESTS_DIR)/../env -I$(RISCV_TESTS_DIR)/common -I../ -I./ 
+COMMON_FILES += $(RISCV_TESTS_DIR)/common/crt.S $(RISCV_TESTS_DIR)/common/syscalls.c 
```

Comment:
What syscalls are we using from RISCV_TESTS? Katie also worked on a crt.S for us, I'm not sure if it's ready yet, but it would be nice to decouple from RISCV_TESTS and use our own stuff...

Reply:
We use the crt.S file for start up, the test.ld for the linker, env/encoding.h for the assembly, and syscalls.c which just has _init which calls main and calls exit afterwards which then writes to the ufinish csr. I'd be open to trying to decouple, this kind of templated approach from riscv-tests has been very easy to port from one testsuite to another so I havn't messed too much with it yet. Maybe could add as an issue that I could resolve after tapeout?

Reply:
Sounds good to me